lib/mjsonwp/mjsonwp.js

```diff
@@ -284,6 +284,11 @@ function buildHandler (app, method, path, spec, driver, isSessCmd) {
       if (spec.command === 'createSession') {
         newSessionId = driverRes[0];
         driverRes = driverRes[1];
+        if (driver.protocol === BaseDriver.DRIVER_PROTOCOL.W3C) {
+          driverRes = {
+            capabilities: driverRes[1],
```

Comment:
I assume there is an error here. It should be replaced with

```
driverRes = {
   capabilities: driverRes
};
```

since we've already assigned `driverRes[1]` before